celery/bin/base.py

```diff
@@ -46,6 +47,11 @@ def OK(self):
     def ERROR(self):
         return self.style("ERROR", fg="red", bold=True)
 
+    @property
+    @functools.lru_cache(None)
```

Comment:
You can use `@cached_property` here.
We have a backport for Python versions which lack it.